tmt/convert.py

```diff
@@ -770,7 +806,8 @@ def read_nitrate_case(testcase, makefile_data=None, general=False):
 
     # Extend bugs detected from Makefile with those linked in Nitrate
     try:
-        data['link'] = makefile_data['link'].copy()
+        if makefile_data is not None:
```

Comment:
if we extend the condition here, maybe we do not need the `try/except` clause?

```suggestion
        if makefile_data is not None and 'link' in makefile_data:
```

?

Reply:
Thanks, fixed